docs/chapters/baseplatform/replacing-packagegroups.rst

```diff
@@ -0,0 +1,57 @@
+:orphan:
+
+.. _replacing-packagegroups:
+
+Replacing package groups by hand-picked components
+==================================================
+
+By default PELUX include components from QT Auto using the package groups feature.
```

Comment:
And the feature is "packagegroups" in yocto terms, without the space.

So please update to say packagegroups throughout the article.